8/execution-environment.yml

```diff
@@ -0,0 +1,23 @@
+----
+
+version: 3
+images:
+  base_image:
+    name: quay.io/fedora/fedora:latest
+dependencies:
+  ansible_core:
+    package_pip: ansible-core==2.15.1
```

Comment:
EE's are Core specific and not "Ansible Community Package" specific. I believe it would be more suitable to include them in the [ansible-community/images](https://github.com/ansible-community/images) repository instead of here.

I understand the reasoning behind having them here to facilitate the building and releasing of both the Community Package and EE together. However, consolidating all image files under one roof (i.e., in [ansible-community/images](https://github.com/ansible-community/images)) would be highly beneficial from a community perspective.

Reply:
by the time this comes out, I think it will be core 2.15.2 that we'd want? (to match the most recent core release?)

Is there a plan to update this ee for every cor release or just...periodically (aka randomly ;-)